CommandLine.c

```diff
@@ -341,7 +341,7 @@ int CommandLine_run(int argc, char** argv) {
    Hashtable* ds = DynamicScreens_new();
 
    Machine* host = Machine_new(ut, flags.userId);
-   ProcessList* pl = ProcessList_new(host, flags.pidMatchList);
+   ProcessTable* pl = ProcessTable_new(host, flags.pidMatchList);
```

Comment:
Maybe rename the local variable as well?
```suggestion
   ProcessTable* pt = ProcessTable_new(host, flags.pidMatchList);
```

Reply:
@Explorer09 good catch, thanks - will do.